### PR TITLE
Upgrade controller dependencies

### PR DESCRIPTION
**Pull request by briantopping**
Update controller dependencies

Hi @lubronzhan, I was working in some other projects and thought that this one would also benefit from the new `vsphere-automation-sdk-go`. Went ahead and upgraded everything else at the same time.

I ran into TLS errors on the test, can you determine if that's a flake or I have more work to do? Thanks! :)

### CODE REVIEW

**Comment by briantopping** [comment]
Returned versions to v0.26.3 and force pushed. Should be clean now to test.

File: pkg/cloudprovider/vsphere/loadbalancer/nsxt_type_converter.go [hunk 1]
```diff
@@ -30,7 +30,6 @@ type nsxtTypeConverter struct {
 
 func newNsxtTypeConverter() *nsxtTypeConverter {
 	converter := bindings.NewTypeConverter()
-	converter.SetMode(bindings.REST)
```
Comment (lubronzhan) [inline pkg/cloudprovider/vsphere/loadbalancer/nsxt_type_converter.go:33]: Do you known whether this has impact

**Comment by lubronzhan** [comment]
Sorry just saw this.
Thanks for the commit. I only have the question on the change in NSX-T broker
/ok-to-test

**Comment by lubronzhan** [comment]
Ok I found it in their release note
```
Updated runtime and libraries to v0.6.0
Update runtime to v0.6.0 by @aagrawal3 in https://github.com/vmware/vsphere-automation-sdk-go/pull/19
Updated lib to v0.6.0 by @aagrawal3 in https://github.com/vmware/vsphere-automation-sdk-go/pull/20
Removed the ConverterMode field of bindings.TypeConverter as it is no longer needed. Also removed SetMode(), Mode() and InRestMode().
```
Thanks for the PR
/lgtm
/approve

**Comment by briantopping** [comment]
Hi @lubronzhan Apologies I missed your inquires here. Thanks for the merge! 👍

**Pull request merged**